src/main/java/org/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,21 +1,21 @@
 package org.bookstore.dto;
 
 import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
-import lombok.Data;
 
-@Data
-public class CreateBookRequestDto {
-    private Long id;
-    @NotBlank
-    private String title;
-    @NotBlank
-    private String author;
-    @NotBlank
-    private String isbn;
-    @PositiveOrZero
-    private BigDecimal price;
-    private String description;
-    private String coverImage;
-}
+public record CreateBookRequestDto(
+        Long id,
+        @NotBlank String title,
+        @NotBlank
+        String author,
```

Comment:
you combined 2 styles here - pls be consistent

either use : `annotation field`
or:
```
annotation
field
```